examples/transcription/websocket/simple_stream.rs

```diff
@@ -27,8 +27,8 @@ async fn main() -> Result<(), DeepgramError> {
         .encoding(Encoding::Linear16)
         .sample_rate(44100)
         .channels(2)
-        // .endpointing(Endpointing::Enabled())
-        .endpointing(Endpointing::CustomValue(300))
+        // .endpointing(Endpointing::Enabled)
```

Comment:
Do we want to leave this here commented out?  Normally I would either uncomment or delete.